tools/quake3/q3map2/bsp.c

```diff
@@ -842,11 +841,30 @@ int BSPMain( int argc, char **argv ){
 			Sys_Printf( "Debug portal surfaces enabled\n" );
 			debugPortals = qtrue;
 		}
-		else if ( !strcmp( argv[ i ], "-bsp" ) ) {
+		else if ( !strcmp( argv[ i ], "-altsplit" ) )
+		{
+			Sys_Printf( "Alternate BSP splitting (by 27) enabled\n" );
+			bspAlternateSplitWeights = qtrue;
+		}
+		else if ( !strcmp( argv[ i ], "-deep" ) )
+		{
+			Sys_Printf( "Deep BSP tree generation enabled\n" );
+			deepBSP = qtrue;
+		}
```

Comment:
Do a more complete pass in netradiant q3map2 to make sure everything related to deepBSP is caught, or remove as well?

Reply:
I re-read each commit entirely side by side and yes some stuff was missing, what was missing is now imported too.